web-console/4.0-designs/storage/Clone/clone.md

```diff
@@ -0,0 +1,55 @@
+# Clone - PVC/PV
+
+## Main use cases
+Create clone (from where PVC/snapshot is)
+All other List, View, Delete, Modify actions are that of a PV/PVC
+
+#### Notes:
+- Alpha support in Kubernetes v1.15
+- critical for stateful workloads, especially databases
+- similar behavior to creating a snapshot
+- From user’s perspective, a clone is another PVC/PV; however contents of a clone is that its PV is populated from contents of a PV at creation time.
+- After creation, the clone behaves just like any other PV.
+- clone using create volume from existing snapshot or existing PV
+- Cloning only support with CSI drivers (and not for in-tree or Flex drivers)
+- To enable the Kubernetes Volume Cloning feature 
+- enabled via the PersistentVolumeClaim.DataSource.Kind field
+ 
+## User Stories
+As a Developer, I want to be able to create a clone of a PVC.
```

Comment:
Should we have another user story around cloning a PV?

Reply:
+1 @lizsurette